src/Rings/mpoly-ideals.jl

```diff
@@ -10,9 +10,8 @@ export ngens, gens
 @doc Markdown.doc"""
     ideal(g::Array{T, 1}) where {T <: MPolyElem}
 
-Create the ideal generated by the polynomials in `g`. 
-
-# Examples
+Creates the ideal generated by the polynomials in `g`. 
+###### Example
```

Comment:
The Julia documentation suggest that docstrings should uniformly use `# Examples` to section off examples (even if it just a single example, the plural is kept). I suggest we keep following that guidelines (it has the advantage that one does not have to remember to add/remove an "s" in a section when adding/removing examples)

Reply:
Sorry, I must have done something wrong while merging the code. I will fix it.